src/Printer_CartridgeInfo.php

```diff
@@ -92,20 +97,30 @@ public function showForPrinter(Printer $printer)
             }
 
             if (is_numeric($value)) {
-                $bar_color = 'green';
                 $progressbar_data = [
-                    'percent'      => $value,
-                    'percent_text' => $value,
-                    'color'        => $bar_color,
-                    'text'         => ''
+                    'percent'           => $value,
+                    'percent_text'      => $value,
+                    'background-color'  => $bar_color,
+                    'text-color'        => $text_color,
+                    'text'              => ''
                 ];
 
-                $out = "{$progressbar_data['text']}<div class='center' style='background-color: #ffffff; width: 100%;
-                     border: 1px solid #9BA563; position: relative;' >";
-                $out .= "<div style='position:absolute;'>&nbsp;{$progressbar_data['percent_text']}%</div>";
-                $out .= "<div class='center' style='background-color: {$progressbar_data['color']};
-                     width: {$progressbar_data['percent']}%; height: 12px' ></div>";
-                $out .= "</div>";
+
+                $out = <<<HTML
+                    <span class='text-nowrap'>
+                    {$progressbar_data['text']}
+                    </span>
+                    <div class="progress" style="height: 16px">
+                        <div class="progress-bar progress-bar-striped" role="progressbar"
+                            style="width: {$progressbar_data['percent']}%; background-color:
+                            {$progressbar_data['background-color']}; color: {$progressbar_data['text-color']};"
+                            aria-valuenow="{$progressbar_data['percent']}"
+                            aria-valuemin="0" aria-valuemax="100">
+                            {$progressbar_data['percent_text']}%
+                        </div>
+
+                    </div>
+                HTML;
```

Comment:
See https://savannah.gnu.org/bugs/?62158
```suggestion
HTML;
```